docs/source/tutorials/pasha/scripts/launch_pasha_nasbench201.py

```diff
@@ -63,84 +60,69 @@ def load_nb201(dataset_name, nb201_random_seed):
 
 
 def run_experiment(
-    dataset_name,
-    random_seed,
-    nb201_random_seed,
-    hpo_approach,
-    reduction_factor=None,
-    rung_system_kwargs={"ranking_criterion": "soft_ranking", "epsilon": 0.025},
-):
+        dataset_name,
+        random_seed,
+        nb201_random_seed,
+        hpo_approach):
     """
     Function to run a NASBench201 experiment. It is similar to the NASBench201 example script
     in syne-tune but extended to make it simple to run our experiments.
 
     :param dataset_name: one of 'cifar10', 'cifar100', 'ImageNet16-120'
-    :param random_seed: e.g. 31415927
+    :param random_seed: e.g. 1
     :param nb201_random_seed: one of 0, 1, 2
     :param hpo_approach: one of 'pasha', 'pasha-bo'
-    :param reduction_factor: by default None (resulting in using the default value 3)
-    :param rung_system_kwargs: dictionary of ranking criterion (str) and epsilon or epsilon scaling (both float)
     :return: tuner.name
     """
 
     logging.getLogger().setLevel(logging.WARNING)
 
-    default_params = nasbench201_default_params({"backend": "simulated"})
-    benchmark = nasbench201_benchmark(default_params)
-    # benchmark must be tabulated to support simulation
-    assert benchmark.get("supports_simulated", False)
-    mode = benchmark["mode"]
-    metric = benchmark["metric"]
-    blackbox_name = benchmark.get("blackbox_name")
-    # NASBench201 is a blackbox from the repository
-    assert blackbox_name is not None
-
-    config_space = benchmark["config_space"]
+    benchmark = nas201_benchmark(dataset_name)
 
     # simulator backend specialized to tabulated blackboxes
+    max_resource_attr = benchmark.max_resource_attr
     trial_backend = BlackboxRepositoryBackend(
-        blackbox_name=blackbox_name,
-        elapsed_time_attr=benchmark["elapsed_time_attr"],
+        blackbox_name=benchmark.blackbox_name,
+        elapsed_time_attr=benchmark.elapsed_time_attr,
+        max_resource_attr=max_resource_attr,
         dataset=dataset_name,
         seed=nb201_random_seed,
     )
 
-    # set logging of the simulator backend to WARNING level
-    logging.getLogger("syne_tune.backend.simulator_backend.simulator_backend").setLevel(
-        logging.WARNING
+    resource_attr = next(iter(trial_backend.blackbox.fidelity_space.keys()))
+    max_resource_level = int(max(trial_backend.blackbox.fidelity_values))
+    config_space = dict(
+        trial_backend.blackbox.configuration_space,
+        **{max_resource_attr: max_resource_level},
     )
 
-    if not reduction_factor:
-        reduction_factor = default_params["reduction_factor"]
+    # set logging of the simulator backend to WARNING level
+    logging.getLogger("syne_tune.backend.simulator_backend.simulator_backend").setLevel(
+        logging.WARNING)
 
     if hpo_approach == "pasha":
         scheduler = baselines_dict["PASHA"](
```

Comment:
Could you please clean this up? You are using `baselines_dict` here, but `HyperbandScheduler` below?

Reply:
I've now cleaned this up too